### PR TITLE
Update Setup.md

### PR DESCRIPTION
**Pull request by Huddie**
Slight confusion recently with entitlements, simulators vs physical devices etc.
This update of the setup guide will hopefully remedy this

Some minor wording updates as well

### CODE REVIEW

**Comment by wayni208** [comment]
I’m at work (on break) but would like to comment and offer some suggestions based on my observations installing and setting up the repo. I can do them later this afternoon. Would that be ok?

**Comment by Huddie** [comment]
@wayni208 Sure! sounds awesome!

**Comment by wayni208** [comment]
Here are some changes I would suggest. 

```
## 1. Cloning the repo
1. Create or navigation to a folder/directory where you’ll store the GitHawk project.

	Example: `mkdir ./desktop/GitHawkApp/`

2. In terminal clone the repo using: `git clone https://github.com/<YourGithubUsername>/GitHawk.git`
```

I would correct "navigation" to read "navigate" in 1. I would change 2 to say:

```
2. Open a terminal in the directory you just created or have selected to store the GitHawk project. In terminal, clone the repo using: `git clone https://github.com/<YourGithubUsername>/GitHawk.git`
```

In order to create "Copy of Freetime-AppCenter" the workspace must be open, allowing access to the schema. This is most of what I came here to say.

```
To add the Client ID and Client Secret to the App, follow these steps:

1. In Xcode, go to `Product` (in the Menu bar) > `Scheme` > `Manage Schemes...`
2. Select `Freetime-AppCenter` and click the cog.
3. Click on `Duplicate`.
4. Uncheck the `Shared` checkbox.
5. Select `Copy of Freetime-AppCenter` and click on `Edit...`
6. Go to `Run` > `Arguments`
7. Add your Client ID (`GITHUB_CLIENT_ID` as key) and Client Secret (`GITHUB_CLIENT_SECRET`) to the Environment Variables.
```

I would change this section like this:

```
To add the Client ID and Client Secret to the App, follow these steps:

1. In the directory you created, open the {YOUR_DIR}/GitHawk/Freetime.xcworkspace file.
2. In Xcode, go to `Product` (in the Menu bar) > `Scheme` > `Manage Schemes...`
3. Select `Freetime-AppCenter` and click the cog.
4. Click on `Duplicate`.
5. Select `Copy of Freetime-AppCenter` and uncheck "Shared" checkbox. 
6. With 'Copy of Freetime-Appcenter' selected, click on `Edit...`
6. Go to `Run` > `Arguments`
7. Add your Client ID (`GITHUB_CLIENT_ID` as key) and Client Secret (`GITHUB_CLIENT_SECRET`) to the Environment Variables.
```

Also, I am using Xcode 10 and when I duplicate the schema it immediately goes into edit mode. "Shared" is disabled by default. 

My issues were after I set up for Code Signing. Perhaps a note in this portion to let the contributor know that project, extension, and plist files will be out of sync and to avoid committing them would be appropriate? While it might be convenient to place these files in .gitignore I have noticed on a recent PR that the project file was updated/altered to account for a new class.

**Comment by Huddie** [comment]
I shall review and update! Thanks so much!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Setup.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-This guide will walk through the setup process required in order to get the GitHawk app running properly.
+This guide will walk through the setup process required in order to get the GitHawk app up and running.
 
 ## 1. Cloning the repo
 1. Create or navigation to a folder/directory where you’ll store the GitHawk project.
 
 	Example: `mkdir ./desktop/GitHawkApp/`
 
-2. In terminal clone the repo using: `git clone https://github.com/<YourGithubName>/GitHawk.git`
+2. In terminal clone the repo using: `git clone https://github.com/<YourGithubUsername>/GitHawk.git`
```
Comment (BasThomas) [inline Setup.md]: We are missing some indication to fork first, I think. :)

**Comment by Huddie** [comment]
@wayni208 @BasThomas Review Request

File: Setup.md [hunk 2]
```diff
@@ -33,24 +36,34 @@ You will be redirected to the application page where you can access your Client
...
+>   Checkpoint: At this point you should be able to successfully build the app on the simulator
+
+🎉 The project is officially set up! 🎉 
+
+At this point you can begin making changes and testing on your simulator.
+If you would like to test on a physical device, one more step remains.
+
 ## 4. Setting up code-signing (With automatic code-signing)
 
+> ⚠️ **1. This step is only if you would like to test/run the app on your physical device 2. This step changes the bundleID and groupID, keep in mind when you commit to leave these changes out**
```
Comment (wayni208) [inline Setup.md]: Did you intend for this line to contain both numbers in the list?

Comment (Huddie) [reply Setup.md]: It’s a block quote. So it doesn’t work as a list I think.
I think it’s okay

Comment (wayni208) [reply Setup.md]: It looks great!

Comment (BasThomas) [reply Setup.md]: Yeah that doesn't seem to work indeed. Should be OK though :)

**Pull request merged**